aikau/src/main/resources/alfresco/notifications/templates/AlfNotification.html

```diff
@@ -1,5 +1,6 @@
 <div class="alfresco-notifications-AlfNotification">
    <div class="alfresco-notifications-AlfNotification__container">
+      <div class="alfresco-notifications-AlfNotification__close" data-dojo-attach-event="click:_onCloseClick">&times;</div>
```

Comment:
For the record, we discussed this offline and I don't like the pattern (personal preference), but there's nothing technically wrong with it and it's consistent with Aikau patterns used elsewhere.